ckanext/ytp/comments/tests/conftest.py

```diff
@@ -0,0 +1,152 @@
+# encoding: utf-8
+
+import os
+import six
+from datetime import datetime as dt
+
+import pytest
+import factory
+from faker import Faker
```

Comment:
This won't work on CKAN 2.8, need to either wrap this in a check or drop CKAN 2.8 support.